packages/libs/avector/macros/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "avector-macros"
```

Comment:
What is the purpose of this crate?

Reply:
removed in [6e55acc](https://github.com/hashintel/hash/pull/1399/commits/6e55accf86ba04894b080d54ba8a88f23655c043)